bescort.lic

```diff
@@ -1462,9 +1464,22 @@ class Bescort
 
     case bput('go ferry', 'The Captain gives you a little nod', 'You hand him', 'The ferry has just pulled away from the dock', 'There is no ferry here to go aboard', 'Come back when you can afford the fare')
     when 'You hand him', 'The Captain gives you a little nod'
-      hide?
-      waitfor 'reaches the dock and its crew ties the ferry off'
-      move 'go dock'
+      # hide?
```

Comment:
Why was this removed?

Reply:
Adding this back in as an option if they have bescort_hide: true